OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -178,6 +195,21 @@ open class OptimoveConfigBuilder: NSObject {
             Logger.error(error.localizedDescription)
         }
 
+        return self
+    } 
+
+    @discardableResult public func setCredentials(
```

Comment:
does this need to be public? customers should not be doing builder.setCredentials() -- they either proovide them immediately or Optimove.setCredentials

Reply:
Hmm, I'm not sure; seems to work when I remove the `public` keyword. @eligutovsky would that break something I am not aware of?